torch/testing/_internal/common_methods_invocations.py

```diff
@@ -14840,6 +14840,9 @@ def error_inputs_mean(op_info, device, **kwargs):
                        # RuntimeError: "index_select_cuda" not implemented for 'ComplexHalf'
                        DecorateInfo(unittest.expectedFailure, 'TestCommon', 'test_python_reference_meta_functions',
                                     dtypes=(torch.chalf,),),
+                       # AssertionError: Tensor-likes are not close!
+                       DecorateInfo(unitted.expectedFailure, 'TestUnaryUfuncs', 'test_reference_numerics_large',
+                                    device_type='cuda', dtypes=(torch.chalf,),),
```

Comment:
Maybe relax tolerance? Would that help?

Eg.
https://github.com/pytorch/pytorch/blob/6fa20bdfe85fe6b92744168edb1495c8e7545f9a/torch/testing/_internal/common_methods_invocations.py#L9700

Reply:
I think it will be the better approach :) I'm giving that a try! Will update it soon.